xml/System.Data.SqlClient/SqlCommand.xml

```diff
@@ -1703,7 +1718,7 @@ SELECT * FROM dbo.Customers WHERE CustomerID = @CustomerID
  A value of 0 indicates no limit (an attempt to execute a command will wait indefinitely).  
   
 > [!NOTE]
->  The <xref:System.Data.SqlClient.SqlCommand.CommandTimeout%2A> property will be ignored during asynchronous method calls such as <xref:System.Data.SqlClient.SqlCommand.BeginExecuteReader%2A>.  
+>  The <xref:System.Data.SqlClient.SqlCommand.CommandTimeout%2A> property will be ignored during old-style asynchronous method calls such as <xref:System.Data.SqlClient.SqlCommand.BeginExecuteReader%2A>. It will be honored by the newer async methods such as <xref:System.Data.SqlClient.SqlCommand.ExecuteReaderAsync%2A>.
```

Comment:
How someone determines whether the method is old-style or not? Is it by the presence of Async on the method name?

/cc @BillWagner

Reply:
The terms we've used in the async docs are "Task Asynchronous Programming" (TAP) and "Asynchronous Programming Model" (APM).  See [this section](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/interop-with-other-asynchronous-patterns-and-types#tasks-and-the-asynchronous-programming-model-apm). 

I agree that "APM" is not very descriptive, but it might be better than "old style" and "new style"

Reply:
Will modify accordingly, thanks.

Reply:
Other than this, the changes look good.

Reply:
Fixed.